mw-webapp/src/utils/markdown/renderMarkdown.tsx

```diff
@@ -8,9 +8,12 @@ const customComponents: Components = {
    * Custom anchor element for markdown
    */
   a: ({children, ...params}) => {
+    const isExternalLink = params.href?.startsWith('http') || params.href?.startsWith('www');
```

Comment:
1. In this PR need to fix all links (read more in issue description and comment)
2. I think we should open in new tab all md links, not only those that starts with "http" and "www". What do you think?
3. Do we have links with http? Could you please tell me where) It's not recommended to use http  because of security

Reply:
let's skip check on prefixes and protocols. All md links should opened in new tab

Reply:
```
Do we have links with http? Could you please tell me where) It's not recommended to use http because of security
```
this method startsWith('http') will be work with 'http' and 'https' links if link start with word http

Reply:
fixed